tools/codegen/config/config_model.go

```diff
@@ -36,11 +36,12 @@ type SchemaOptions struct {
 }
 
 type Override struct {
-	Computability *Computability `yaml:"computability,omitempty"`
-	Sensitive     *bool          `yaml:"sensitive"`
-	Description   string         `yaml:"description"`
-	PlanModifiers []PlanModifier `yaml:"plan_modifiers"`
-	Validators    []Validator    `yaml:"validators"`
+	Computability     *Computability `yaml:"computability,omitempty"`
+	Sensitive         *bool          `yaml:"sensitive"`
+	IncludeJSONUpdate *bool          `yaml:"include_json_update"`
```

Comment:
The name seems too generic for what it actually does. Would `IncludeNullOnUpdate` be more clear?